packages/adblocker/src/filters/network.ts

```diff
@@ -400,6 +400,20 @@ function compileRegex(
   return new RegExp(filter);
 }
 
+function findLastIndexOfUnescapedCharacter(text: string, character: string) {
+  let lastIndex = text.lastIndexOf(character);
+
+  if (lastIndex === -1) {
+    return -1;
+  }
+
+  while (text.charCodeAt(lastIndex - 1) === 92 /* '\\' */) {
+    lastIndex = text.slice(0, lastIndex).lastIndexOf(character);
```

Comment:
Isn’t it possible to do something like: text.lastIndexOf(character, lastIndex - 1) in order to avoid slicing.

Reply:
Sorry, it should be better definitely. I wasn't aware of the position argument difference compared to `indexOf`.